Remove misleading comment

TimothyGu opened the pull request:
If the line is uncommented, the base URL is overridden immediately below any and wouldn't work.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@alexandair : Thanks for your contribution to the Azure documentation! The author, @iainfoulds, has been notified to review your proposed change.

iainfoulds commented:
Thanks for the updates, @alexandair!

@tysonn : This PR can be merged.

TimothyGu committed:
Remove misleading comment

If the line is uncommented, the base URL is overridden immediately below any and wouldn't work.
